src/PureScript/CST/Utils/ModuleGraph.purs

```diff
@@ -0,0 +1,87 @@
+module PureScript.CST.Utils.ModuleGraph
```

Comment:
I don't know if this _needs_ to be in a `Utils` namespace. Usually, I think of Utils as being kind of like Internal. A grab bag of useful stuff that's used to implement stuff in the library but not necessarily for public consumption.

Reply:
Okay, that sounds reasonable to me, I'll move it